project-base/composer.json

```diff
@@ -79,8 +79,8 @@
         "snc/redis-bundle": "^2.1.8",
         "stof/doctrine-extensions-bundle": "^1.3.0",
         "symfony/assetic-bundle": "^2.8.2",
-        "symfony/monolog-bundle": "^3.1.2",
-        "symfony/swiftmailer-bundle": "^3.2.2",
+        "symfony/monolog-bundle": "^3.4",
+        "symfony/swiftmailer-bundle": "^3.4",
```

Comment:
WTF mr. @TomasLudvik  https://packagist.org/packages/symfony/monolog-bundle

Reply:
by the way in monorepo composer it is not changed .....

Reply:
I changed it to latest released version in all composer.json files